data/org.freedesktop.impl.portal.ScreenCast.xml

```diff
@@ -81,6 +81,37 @@
               Whether to allow selecting multiple sources. Default is no.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>restore_data as</term>
```

Comment:
Is this meant to be the same thing as the `restore_data v` in the `Response`?

How sure are we that an array of UTF-8 strings is the right type?